experimental/telemetry/doc.go

```diff
@@ -15,4 +15,37 @@
 // If you want to use your own data plane URL, add also this line and
 // make sure the MM_RUDDER_DATA_PLANE_URL environment variable is set.
 //   LDFLAGS += -X "github.com/mattermost/mattermost-plugin-api/experimental/telemetry.rudderDataPlaneURL=$(MM_RUDDER_DATA_PLANE_URL)"
+//
+// In order to use it you should:
+// 1. Add the new fields to the plugin
+//   type Plugin struct {
+//     plugin.MattermostPlugin
+//     ...
+//     telemetryClient telemetry.Client
+//     tracker         telemetry.Tracker
+//   }
+// 2. Start the telemetry client on plugin activate:
```

Comment:
```suggestion
// 2. Start the telemetry client on plugin activate
```

Reply:
I prefer not having the `:` but will leave up to you.  The only ask is that all numbered items are consistant